index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#"><h2 class="welcome">Welcome</h2></a>
+            <div class="nav-links">
+                <i id="menu" class="material-symbols-outlined">menu</i>
+            </div>
+        </nav>
+    </header>
+
+        <section class="headline">
+            <h1 class="black">Hey there. I’m Desmond.<br>
+                <span id="green"> I’m a software developer</span></h1>
+                <p class="text" id="ptext"> 
+                    I can help you build a product ,<br> feature or website Look through<br> some of my work and experience!<br> If you like what you see and have<br> a project you need coded,<br>
+                   don’t hestiate to contact me.
+                </p>
+            
+        <ul class="socialmedia-icons">
+                <li>
+                    <a href="#"><img src="images/Vector.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Linkedinicon.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vectora.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vectorb.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vectorc.jpg" alt=""></a>
+                </li>
+            </ul>
```

Comment:
Kindly endeavour to indent your codes properly as it is one of the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and will help make your code more readable.

The opening `<ul>` tag isn't aligned with its closing tag